data/task.py

```diff
@@ -84,19 +84,21 @@ def status(self):
   def status(self, value):
     """Sets status as State."""
     if self._validate_type(int, value, 'status'):
-      if value == 0:
-        status = State.UNUSED_STATUS
-      elif value == 1:
-        status = State.PENDING
+      if value == 1:
```

Comment:
Can we replace this by if 1 <= value <=6: state = value

Reply:
Done